config/registry/aws/modules/aws-k8s-service.yaml

```diff
@@ -105,6 +105,16 @@ inputs:
     validator: str(required=False)
     description: Use if liveness probe != readiness probe
     default: null
+  - name: sticky_session
+    user_facing: true
+    validator: bool(required=False)
+    description: Use [sticky sessions](https://stackoverflow.com/questions/10494431/sticky-and-non-sticky-sessions) via cookies for your service (first request will send you a cookie called opta_cookie which you should add on future requests).
+    default: false
+  - name: sticky_session_max_age
+    user_facing: true
+    validator: int(required=False)
+    description: If the sticky session is enabled, how long should the cookie last?
+    default: 86400
```

Comment:
Is the cookie renewed, or is it expired? From the canvas point of view it may be an issue if suddenly everybody gets booted out of the joint cookie session...

Reply:
great question-- a new cookie is created and silently inserted. Canvas has the cookie expiration time so they should be able to work with a graceful replace.